packages/react-native-editor/bin/generate-pot-files.sh

```diff
@@ -0,0 +1,254 @@
+#   generate-pot-files.sh --path <PATH> domain-plugin-1 <PLUGIN-1_SOURCE_PATH> domain-plugin-2 <PLUGIN-2_SOURCE_PATH>
+#
+
+# Exit if any command fails
+set -euo pipefail
+
+# Get arguments
+while test $# -gt 0; do
+  case "$1" in
+    -h|--help)
+      echo "options:"
+      echo "-h, --help                              show brief help"
+      echo "-w, --skip-upgrade-wp-cli               skip WP-CLI upgrade"
+      echo "-p, --path                              use local path for generating files"
+      echo "-d, --debug                             print extra info for debugging"
+      exit 0
+      ;;
+    -w|--skip-upgrade-wp-cli*)
+      shift
+      SKIP_UPGRADE_WP_CLI='true'
+      ;;
+    -p|--path*)
+      shift
+      LOCAL_PATH=$1
+      shift
+      ;;
+    -d|--debug*)
+      shift
+      DEBUG='true'
+      ;;
+    *)
+      break
+      ;;
+  esac
+done
+
+# Functions
+function join_by { local IFS="$1"; shift; echo "$*"; }
+
+function error() {
+  echo -e "\033[0;31m$1\033[0m"
+  exit 1
+}
+
+function setup_wp_cli() {
+  local cli_path="$SCRIPT_DIR/wp-cli.phar"
+
+  # Install WP-CLI command
+  if [[ ! -f "$cli_path" ]]; then
+    echo -e "\n\033[1mInstalling WP-CLI\033[0m"
+    curl -Ls https://raw.githubusercontent.com/wp-cli/builds/gh-pages/phar/wp-cli.phar -o $cli_path
+    chmod +x $cli_path
+  fi
+
+  # Upgrade WP-CLI command
+  if [[ -z "${SKIP_UPGRADE_WP_CLI:-}" ]]; then
+    echo -e "\n\033[1mUpgrading WP-CLI\033[0m"
+    $WP_CLI cli update --nightly --yes
+    $WP_CLI --info
+  fi
+}
+
+function generate_bundles() {
+  local entry_file=$1
+  local android_bundle_output=$2
+  local android_sourcemap_output=$3
+  local ios_bundle_output=$4
+  local ios_sourcemap_output=$5
+
+  echo -e "\n\033[1mGenerate Android JS bundle\033[0m"
+  $BUNDLE_CLI --platform android --dev false --entry-file "$entry_file" --bundle-output "$android_bundle_output" --sourcemap-output "$android_sourcemap_output"
+  
+  echo -e "\n\033[1mGenerate iOS JS bundle\033[0m"
+  $BUNDLE_CLI --platform ios --dev false --entry-file "$entry_file" --bundle-output "$ios_bundle_output" --sourcemap-output "$ios_sourcemap_output"
+}
+
+function extract_source_from_sourcemap_file() {
+  local map_file=$1
+  local target_path=$2
+
+  mkdir -p $target_path
+
+  echo -e "\n\033[1mExtracting source files from \"$map_file\" source map file\033[0m"
+  node $SCRIPT_DIR/extract-files-from-sourcemap.js $map_file $target_path
+}
+
+function make_pot () {
+  local source=$1
+  local arguments=$2
+  local makepot_command="$WP_CLI i18n make-pot"
+
+  # In order to detect parse errors, we need to use the "--debug" option, otherwise "make-pot" command doesn't output errors.
+  local full_command="$makepot_command $source --debug $arguments"
+  if [[ -z ${DEBUG:-} ]]; then
+    # When DEBUG flag is not enabled, we only print the parse errors.
+    $full_command 2> >(grep 'Could not parse file' | sed 's/Debug[[:space:]][(]make-pot[)]/Warning/g' >&2)
```

Comment:
The parse errors are logged by the i18n command to `stderr` as:
`Debug (parse-pot): Could not parse file <FILE>`

This line substitutes the content of `stderr` by only including the parse errors entries and replacing the title `Debug (parse-pot)` with `Warning`.